nxdrive/engine/processor.py

```diff
@@ -227,7 +227,12 @@ def _execute(self) -> None:
                     continue
 
                 parent_path = doc_pair.local_parent_path
-
+                if (
+                    doc_pair.local_state == "direct"
+                    and str(parent_path) == "NOT_AVAILABLE"
+                ):
+                    self.engine.queue_manager.push(doc_pair)
```

Comment:
Add a debug log line, it will help following the process when we need to debug things.